packages/docs/components/select.md

```diff
@@ -1,653 +1,42 @@
-::: slot nimatron-all
```

Comment:
Did this get moved?

Reply:
we no longer use the `nimatron-all` slot, so It has been removed. Currently pages have links to legacy docs if you need to reference old content.

Reply:
Sorry, I meant `select.md` altogether. This file seems to be deleted, not moved.